airflow-core/docs/authoring-and-scheduling/timetable.rst

```diff
@@ -316,13 +316,14 @@ and ``data_interval_end`` (and the legacy ``execution_date``) are the same; the
 triggered.
 
 For a data interval timetable, the value of ``data_interval_start`` and ``data_interval_end`` (and legacy
-``execution_date``) are different. ``data_interval_start`` is the time when a DAG run is triggered and
-``data_interval_end`` is the end of the interval.
+``execution_date``) are different. ``data_interval_end`` is the time when a DAG run is triggered, while
+``data_interval_start`` and ``execution_date`` are the start of the interval.
```

Comment:
That is not correct for Airflow 3.0++ - execution date is not existing since 3.0 anymore.